tests/sentry/grouping/snapshots/test_fingerprinting/test_event_hash_variant/fingerprint_exception_type_and_sdk.pysnap

```diff
@@ -0,0 +1,38 @@
+---
+created: '2024-01-10T00:09:15.538545Z'
+creator: sentry
+source: tests/sentry/grouping/test_fingerprinting.py
+---
+config:
+  rules:
+  - attributes: {}
+    fingerprint:
+    - chunkloaderror
+    matchers:
+    - - sdk
+      - sentry.javascript.nextjs
+    - - type
+      - ChunkLoadError
+  version: 1
+fingerprint:
+- chunkloaderror
+title: 'ChunkLoadError: ChunkLoadError something something...'
+variants:
+  app:
+    component:
+      contributes: false
+      hint: custom fingerprint takes precedence
+    type: component
+  custom-fingerprint:
+    client_values:
+    - my-route
+    - '{{ default }}'
+    matched_rule: sdk:"sentry.javascript.nextjs" type:"ChunkLoadError" -> "chunkloaderror"
```

Comment:
This shows that the customer's fingerprint has matched.

Reply:
yeah, in this PR it _is_  a customer's rule. This PR only adds SDK matcher, it works independently of the other PR introducing built-in rules. I think I'm gonna change the example to something else to avoid this confusion.